src/cpu.cpp

```diff
@@ -847,4 +851,12 @@ void set_kmp_blocktime(int time_ms)
 #endif
 }
 
+void set_denormals_zero()
+{
+#if defined(__SSE3__)
+    _MM_SET_FLUSH_ZERO_MODE(_MM_FLUSH_ZERO_ON);
+    _MM_SET_DENORMALS_ZERO_MODE(_MM_DENORMALS_ZERO_ON);
+#endif
+}
```

Comment:
please provide a parameter for controlling the Flush Zero and Denorm Zero mode, so that caller could restore the previous state.
It would be more convenient to have a getter function too.

Reply:
I agree to turn on/off FTZ and DAZ mode through parameters as your suggestion.
I'll add a getter and update it so that it can be set through parameters.

However, I am curious about the advantages of disabling FTZ or DAZ during the inference.